docs/UserGuide.md

```diff
@@ -546,8 +529,8 @@ Parameters:
 **:exclamation: Important:**<br>
 
 * If the parameters are not provided, <b><code>edit INDEX</code> operates the same way as <code>view INDEX</code></b>, leading you to the profile page of the person at index <code>INDEX</code> in the address book. 
-* If you run the same edit command multiple times consecutively (resulting in no visible change after the first run), the undo command will not be able to revert the data back to the original state. <br>
-  This is because undo can only undo the last command, even if the command made no visible changes.
+* If you run the **same edit command multiple times consecutively** (resulting in no visible change after the first run), the undo command **will not** be able to revert the data back to the original state. <br>
+  This is because undo can only undo the last command, **even if the command made no visible changes**.
```

Comment:
In the sentence "this is because...", can we make the "undo" appear in code format, like the code `undo` thing HAHAHA, same for a similar sentence in sort i think

Reply:
can!